lektor/admin/modules/serve.py

```diff
@@ -1,15 +1,17 @@
-import os
 import mimetypes
+import os
 import posixpath
 from cStringIO import StringIO
+from werkzeug.exceptions import NotFound
 from zlib import adler32
 
-from flask import Blueprint, current_app, abort, Response, request, \
-     render_template
+from flask import (Blueprint, Response, abort, current_app, render_template,
+    request)
```

Comment:
this syntax must require `absolute_import` to work correctly. See this [docs](https://docs.python.org/2.7/library/__future__.html) and [pep](https://www.python.org/dev/peps/pep-0328/).